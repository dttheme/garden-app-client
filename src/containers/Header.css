.header {
  margin: 0;
  padding: 0;
  height: 10em;
  background-position: center;
  background-size: cover;
  background-image: url("../images/DIRT2.jpg") ;
}
h1 {
  margin-top: 0;
  color: #B03D32;
}
nav ul li {
  display: inline-block;
  padding-right: 1em;
}
nav a {
  color: #277584;
}
