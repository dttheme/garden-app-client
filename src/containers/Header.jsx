import React, { Component } from 'react';
import { Link } from 'react-router-dom';

// import './Header.css';

class Header extends Component {
  render() {
    return (
        <div className="header">
          <h1>Garden App</h1>
          <nav>
            <ul>
              <li>
                <Link to="/">Home</Link>
              </li>
              <li>
                <Link to="/signup">Sign Up</Link>
              </li>
              <li>
                <Link to="/login">Login</Link>
              </li>
              <li>
                <Link to="/dashboard">Dashboard</Link>
              </li>
              <li>
                <Link to="/garden">Garden</Link>
              </li>
            </ul>
          </nav>
        </div>
    );
  }
}

export default Header;
