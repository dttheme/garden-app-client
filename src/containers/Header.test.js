import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('<Header />', () => {
  it('Should render without crashing', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot()
  });
});
