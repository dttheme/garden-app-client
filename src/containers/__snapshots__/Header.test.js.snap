// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should render without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="dark-gray bg-dark-green fl fixed z-9999 w-100 h-auto pt3 pb3 pl4 pr4 bb bw3 b--green"
>
        <h1
                className="dark-gray f2 f1-ns fw9 pa0 ma0 dib tracked"
        >
                Sprout Bud
        </h1>
        <nav
                className="fr"
        >
                <ul>
                        <li
                                className="dib f5 fw9 tracked"
                        >
                                <Link
                                        className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                        replace={false}
                                        to="/"
                                >
                                        Home
                                </Link>
                        </li>
                        <li
                                className="dib f5 fw9 tracked"
                        >
                                <Link
                                        className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                        replace={false}
                                        to="/signup"
                                >
                                        Sign Up
                                </Link>
                        </li>
                        <li
                                className="dib f5 fw9 tracked"
                        >
                                <Link
                                        className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                        replace={false}
                                        to="/login"
                                >
                                        Login
                                </Link>
                        </li>
                </ul>
        </nav>
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="dark-gray f2 f1-ns fw9 pa0 ma0 dib tracked"
>
            Sprout Bud
</h1>,
          <nav
            className="fr"
>
            <ul>
                        <li
                                    className="dib f5 fw9 tracked"
                        >
                                    <Link
                                                className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                replace={false}
                                                to="/"
                                    >
                                                Home
                                    </Link>
                        </li>
                        <li
                                    className="dib f5 fw9 tracked"
                        >
                                    <Link
                                                className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                replace={false}
                                                to="/signup"
                                    >
                                                Sign Up
                                    </Link>
                        </li>
                        <li
                                    className="dib f5 fw9 tracked"
                        >
                                    <Link
                                                className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                replace={false}
                                                to="/login"
                                    >
                                                Login
                                    </Link>
                        </li>
            </ul>
</nav>,
        ],
        "className": "dark-gray bg-dark-green fl fixed z-9999 w-100 h-auto pt3 pb3 pl4 pr4 bb bw3 b--green",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sprout Bud",
            "className": "dark-gray f2 f1-ns fw9 pa0 ma0 dib tracked",
          },
          "ref": null,
          "rendered": "Sprout Bud",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <ul>
              <li
                            className="dib f5 fw9 tracked"
              >
                            <Link
                                          className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                          replace={false}
                                          to="/"
                            >
                                          Home
                            </Link>
              </li>
              <li
                            className="dib f5 fw9 tracked"
              >
                            <Link
                                          className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                          replace={false}
                                          to="/signup"
                            >
                                          Sign Up
                            </Link>
              </li>
              <li
                            className="dib f5 fw9 tracked"
              >
                            <Link
                                          className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                          replace={false}
                                          to="/login"
                            >
                                          Login
                            </Link>
              </li>
</ul>,
            "className": "fr",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li
                  className="dib f5 fw9 tracked"
>
                  <Link
                                    className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                    replace={false}
                                    to="/"
                  >
                                    Home
                  </Link>
</li>,
                <li
                  className="dib f5 fw9 tracked"
>
                  <Link
                                    className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                    replace={false}
                                    to="/signup"
                  >
                                    Sign Up
                  </Link>
</li>,
                <li
                  className="dib f5 fw9 tracked"
>
                  <Link
                                    className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                    replace={false}
                                    to="/login"
                  >
                                    Login
                  </Link>
</li>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    className="washed-green no-underline underline-hover hover-light-yellow pr4"
                    replace={false}
                    to="/"
>
                    Home
</Link>,
                  "className": "dib f5 fw9 tracked",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Home",
                    "className": "washed-green no-underline underline-hover hover-light-yellow pr4",
                    "replace": false,
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": "Home",
                  "type": [Function],
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    className="washed-green no-underline underline-hover hover-light-yellow pr4"
                    replace={false}
                    to="/signup"
>
                    Sign Up
</Link>,
                  "className": "dib f5 fw9 tracked",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Sign Up",
                    "className": "washed-green no-underline underline-hover hover-light-yellow pr4",
                    "replace": false,
                    "to": "/signup",
                  },
                  "ref": null,
                  "rendered": "Sign Up",
                  "type": [Function],
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    className="washed-green no-underline underline-hover hover-light-yellow pr4"
                    replace={false}
                    to="/login"
>
                    Login
</Link>,
                  "className": "dib f5 fw9 tracked",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Login",
                    "className": "washed-green no-underline underline-hover hover-light-yellow pr4",
                    "replace": false,
                    "to": "/login",
                  },
                  "ref": null,
                  "rendered": "Login",
                  "type": [Function],
                },
                "type": "li",
              },
            ],
            "type": "ul",
          },
          "type": "nav",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="dark-gray bg-dark-green fl fixed z-9999 w-100 h-auto pt3 pb3 pl4 pr4 bb bw3 b--green"
>
          <h1
                    className="dark-gray f2 f1-ns fw9 pa0 ma0 dib tracked"
          >
                    Sprout Bud
          </h1>
          <nav
                    className="fr"
          >
                    <ul>
                              <li
                                        className="dib f5 fw9 tracked"
                              >
                                        <Link
                                                  className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                  replace={false}
                                                  to="/"
                                        >
                                                  Home
                                        </Link>
                              </li>
                              <li
                                        className="dib f5 fw9 tracked"
                              >
                                        <Link
                                                  className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                  replace={false}
                                                  to="/signup"
                                        >
                                                  Sign Up
                                        </Link>
                              </li>
                              <li
                                        className="dib f5 fw9 tracked"
                              >
                                        <Link
                                                  className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                  replace={false}
                                                  to="/login"
                                        >
                                                  Login
                                        </Link>
                              </li>
                    </ul>
          </nav>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              className="dark-gray f2 f1-ns fw9 pa0 ma0 dib tracked"
>
              Sprout Bud
</h1>,
            <nav
              className="fr"
>
              <ul>
                            <li
                                          className="dib f5 fw9 tracked"
                            >
                                          <Link
                                                        className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                        replace={false}
                                                        to="/"
                                          >
                                                        Home
                                          </Link>
                            </li>
                            <li
                                          className="dib f5 fw9 tracked"
                            >
                                          <Link
                                                        className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                        replace={false}
                                                        to="/signup"
                                          >
                                                        Sign Up
                                          </Link>
                            </li>
                            <li
                                          className="dib f5 fw9 tracked"
                            >
                                          <Link
                                                        className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                        replace={false}
                                                        to="/login"
                                          >
                                                        Login
                                          </Link>
                            </li>
              </ul>
</nav>,
          ],
          "className": "dark-gray bg-dark-green fl fixed z-9999 w-100 h-auto pt3 pb3 pl4 pr4 bb bw3 b--green",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Sprout Bud",
              "className": "dark-gray f2 f1-ns fw9 pa0 ma0 dib tracked",
            },
            "ref": null,
            "rendered": "Sprout Bud",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ul>
                <li
                                className="dib f5 fw9 tracked"
                >
                                <Link
                                                className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                replace={false}
                                                to="/"
                                >
                                                Home
                                </Link>
                </li>
                <li
                                className="dib f5 fw9 tracked"
                >
                                <Link
                                                className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                replace={false}
                                                to="/signup"
                                >
                                                Sign Up
                                </Link>
                </li>
                <li
                                className="dib f5 fw9 tracked"
                >
                                <Link
                                                className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                                replace={false}
                                                to="/login"
                                >
                                                Login
                                </Link>
                </li>
</ul>,
              "className": "fr",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li
                    className="dib f5 fw9 tracked"
>
                    <Link
                                        className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                        replace={false}
                                        to="/"
                    >
                                        Home
                    </Link>
</li>,
                  <li
                    className="dib f5 fw9 tracked"
>
                    <Link
                                        className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                        replace={false}
                                        to="/signup"
                    >
                                        Sign Up
                    </Link>
</li>,
                  <li
                    className="dib f5 fw9 tracked"
>
                    <Link
                                        className="washed-green no-underline underline-hover hover-light-yellow pr4"
                                        replace={false}
                                        to="/login"
                    >
                                        Login
                    </Link>
</li>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Link
                      className="washed-green no-underline underline-hover hover-light-yellow pr4"
                      replace={false}
                      to="/"
>
                      Home
</Link>,
                    "className": "dib f5 fw9 tracked",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Home",
                      "className": "washed-green no-underline underline-hover hover-light-yellow pr4",
                      "replace": false,
                      "to": "/",
                    },
                    "ref": null,
                    "rendered": "Home",
                    "type": [Function],
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Link
                      className="washed-green no-underline underline-hover hover-light-yellow pr4"
                      replace={false}
                      to="/signup"
>
                      Sign Up
</Link>,
                    "className": "dib f5 fw9 tracked",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Sign Up",
                      "className": "washed-green no-underline underline-hover hover-light-yellow pr4",
                      "replace": false,
                      "to": "/signup",
                    },
                    "ref": null,
                    "rendered": "Sign Up",
                    "type": [Function],
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Link
                      className="washed-green no-underline underline-hover hover-light-yellow pr4"
                      replace={false}
                      to="/login"
>
                      Login
</Link>,
                    "className": "dib f5 fw9 tracked",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "Login",
                      "className": "washed-green no-underline underline-hover hover-light-yellow pr4",
                      "replace": false,
                      "to": "/login",
                    },
                    "ref": null,
                    "rendered": "Login",
                    "type": [Function],
                  },
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            "type": "nav",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
